scripts/publish.sh

```diff
@@ -36,11 +36,11 @@ if [ -z "$AZURE_DEVOPS_PERSONAL_ACCESS_TOKEN" ]; then
 elif [ -n "$AZURE_DEVOPS_PERSONAL_ACCESS_TOKEN" ]; then
     if [ "$PRODUCTION" = "1" ]; then
         echo "Publishing $RELEASE_CHANNEL release"
-        ./node_modules/.bin/vsce publish "$NEXT_EXTENSION_VERSION" --pat "$AZURE_DEVOPS_PERSONAL_ACCESS_TOKEN"
+        cd packages/vscode && ./node_modules/.bin/vsce publish "$NEXT_EXTENSION_VERSION" --pat "$AZURE_DEVOPS_PERSONAL_ACCESS_TOKEN" && cd ../..
```

Comment:
Is there a reason to move the scripts back to root?

Reply:
yes, for publishing, we need to go in both folders equal times, running everything from the vscode folder leads to a lot of cd statements and `cd ..`. Also git add command has to be run from the root folder.